webserver/webserver/src/main/java/io/helidon/webserver/http1/Http1Connection.java

```diff
@@ -196,6 +196,8 @@ public void handle(Limit limit) throws InterruptedException {
                                 return;
                             }
                         }
+                    } else {
+                        ctx.log(LOGGER, WARNING, "Protocol upgrade for a request with a payload ignored");
```

Comment:
Wouldn't be a debug level better?

Reply:
Maybe, but I would think this is something to be reported more as a warning to indicate that _upgrades are ignored due to an invalid upgrade request_. As DEBUG, it will likely always be unreported. After all, this is a condition that shouldn't really happen, and if it does, then it is nice to know about it.